retry/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v3.2.0
+ * `orElse` callback to calculate a value when the attempts where exhausted.
```

Comment:
```suggestion
 * Introduced `orElse` callback to create a value when the retry attempts are exhausted.
```

Reply:
Hm, yeah, it may be confusing. Maybe `onRetryExhausted` could be used instead?